### PR TITLE
Refactored go-kart code

### PR DESCRIPTION
**Pull request by DamianX**
Nothing to see here, go back to sleep.

### CODE REVIEW

**Comment by robintime** [comment]
Thank you for your service.

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -1193,3 +1193,12 @@
 	target = get_turf(target_loc)
 
 	current_tick = 1
+
+
+/atom/movable/proc/speen()
```
Comment (Exxion) [inline code/game/atoms_movable.dm]: Fuck you

File: code/game/atoms_movable.dm [hunk 2]
```diff
@@ -1193,3 +1193,12 @@
...
+	set waitfor = FALSE
+	var/prev_dir = dir
+	for(var/i in list(1, 4, 2, 8, 1, 4, 2, 8, 1, 4, 2, 8, 1, 4, 2, 8))
```
Comment (Exxion) [inline code/game/atoms_movable.dm]: I know you didn't write this but fuck you even more anyway

File: code/game/atoms_movable.dm (hunk 1 above)
Comment (DamianX) [reply code/game/atoms_movable.dm]: Look, there are things already implementing `spin()`, I was strong-armed into this

**Pull request merged**